### PR TITLE
UIDS-63 update tooltip text prop

### PR DESCRIPTION
**Pull request by AlexAlt**
closes https://github.com/user-interviews/ui-design-system/issues/63

result of https://github.com/user-interviews/rails-server/pull/4678/

### CODE REVIEW

File: src/Popper/Popper.scss [hunk 1]
```diff
@@ -9,7 +9,6 @@
   max-width: 15rem;
   letter-spacing: normal;
   padding: 0.75rem;
-  white-space: pre-line;
```
Comment (AlexAlt) [inline src/Popper/Popper.scss:12]: we don't need this if we can do JSX

File: src/Tooltip/Tooltip.jsx [hunk 2]
```diff
@@ -32,7 +32,7 @@ export default class Tooltip extends Component {
     iconClasses: PropTypes.string,
     placement: PropTypes.string.isRequired,
     strategy: PropTypes.string,
-    text: PropTypes.string.isRequired,
+    text: PropTypes.oneOfType([propTypes.string, propTypes.node]).isRequired,
```
Comment (davidmferris) [inline src/Tooltip/Tooltip.jsx:35]: Might be worth adding a "story" in Tooltip.stories.jsx so we can see how it behaves if you pass in HTML? Lemme know if you want a hand with that.

**Review by rroppolo — APPROVED** [review]
I like Dave's story idea just to help people discover this, but yeah LGTM! 👍

**Review by davidmferris — APPROVED** [review]
Looks good!

**Pull request merged**